### PR TITLE
Remove magic numbers in Lightning stopping_conditions

### PR DESCRIPTION
**Pull request by maliasadi**
### Before submitting

Please complete the following checklist when submitting a PR:

- [ ] All new features must include a unit test.
      If you've fixed a bug or added code that should be tested, add a test to the
      [`tests`](../tests) directory!

- [ ] All new functions and code must be clearly commented and documented.
      If you do make documentation changes, make sure that the docs build and
      render correctly by running `make docs`.

- [ ] Ensure that the test suite passes, by running `make test`.

- [ ] Add a new entry to the `.github/CHANGELOG.md` file, summarizing the
      change, and including a link back to the PR.

- [ ] Ensure that code is properly formatted by running `make format`. 

When all the above are checked, delete everything above the dashed
line and fill in the pull request template.

------------------------------------------------------------------------------------------------------------

**Context:**

**Description of the Change:**

**Benefits:**

**Possible Drawbacks:**

**Related GitHub Issues:**
[sc-74414]

### CODE REVIEW

File: pennylane_lightning/lightning_tensor/lightning_tensor.py [hunk 1]
```diff
@@ -170,11 +170,6 @@ def stopping_condition(op: Operator) -> bool:
     """A function that determines whether or not an operation is supported by the ``mps`` method of ``lightning.tensor``."""
     # TODOs: These thresholds are from ``lightning.qubit`` and should be adjuested based on the benchmarking tests for the MPS
```
Comment (multiphaseCFD) [inline pennylane_lightning/lightning_tensor/lightning_tensor.py:171]: Let's remove line #171 too?

File: pennylane_lightning/lightning_kokkos/lightning_kokkos.py [hunk 2]
```diff
@@ -153,10 +153,6 @@ def stopping_condition(op: Operator) -> bool:
     """A function that determines whether or not an operation is supported by ``lightning.kokkos``."""
     # To avoid building matrices beyond the given thresholds.
```
Comment (multiphaseCFD) [inline pennylane_lightning/lightning_kokkos/lightning_kokkos.py:154]: Let's remove line#154-155 as well?

**Review by multiphaseCFD — CHANGES_REQUESTED** [review]
Thanks @maliasadi . LGTM. Don't forget to add entry to the changelog.

File: pennylane_lightning/lightning_tensor/lightning_tensor.py (hunk 1 above)
Comment (maliasadi) [reply pennylane_lightning/lightning_tensor/lightning_tensor.py:171]: Good catch!

**Review by multiphaseCFD — APPROVED** [review]
Nice one! Thanks @maliasadi !

**Review by AmintorDusko — APPROVED** [review]
Thank you! Much simpler now!

File: pennylane_lightning/lightning_qubit/lightning_qubit.py [hunk 3]
```diff
@@ -170,13 +170,6 @@
 
 def stopping_condition(op: Operator) -> bool:
     """A function that determines whether or not an operation is supported by ``lightning.qubit``."""
-    # To avoid building matrices beyond the given thresholds.
-    # This should reduce runtime overheads for larger systems.
-    if isinstance(op, qml.QFT):
-        return len(op.wires) < 10
```
Comment (LuisAlfredoNu) [inline pennylane_lightning/lightning_qubit/lightning_qubit.py:176]: Looks like you need to update the testing suite because fails some test :sweat_smile:

**Review by LuisAlfredoNu — CHANGES_REQUESTED** [review]

File: pennylane_lightning/lightning_qubit/lightning_qubit.py (hunk 3 above)
Comment (maliasadi) [reply pennylane_lightning/lightning_qubit/lightning_qubit.py:176]: yeah, working on them :smile:

**Review by LuisAlfredoNu — APPROVED** [review]
Thank you @maliasadi 
Nice work! :smile:

**Pull request merged**